roles/identity-management/manage-rh-sso-identities/tasks/process_all_tasks.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: Process Red Hat SSO Identities
+  block:
+    - include_tasks: create_users.yml
+#   - include_tasks: update_users.yml
```

Comment:
Remove commented code.

Reply:
Thanks for pointing that out. This has to do with the comment above (about commenting things out to trigger the right operation). I will remove the comments.